e2e-test/cypress/integration/change_user_password_spec.js

```diff
@@ -1,114 +1,130 @@
-    cy.get("#userPassword-helper-text").contains("Incorrect password");
-
-    // Leave the window
-    cy.get("[data-test=cancel]").click();
-  });
-
-  it("An error is displayed if the new passwords don't match (the user password is not checked)", function() {
-    cy.get("[data-test=edit-user-mstein]").should("be.visible");
-
-    cy.get("[data-test=edit-user-mstein]").click();
-    // Wrong user password entered, but it won't be checked
-    cy.get("[data-test=user-password-textfield] input").type("asdf");
-    cy.get("[data-test=new-password-textfield] input").type("asdf1");
-    cy.get("[data-test=new-password-confirmation-textfield] input").type("asdf2");
-    cy.get("[data-test=submit]").click();
-
-    // The warning "Passwords don't match" is displayed
-    cy.get("#newPasswordConfirmation-helper-text").contains("Passwords don't match");
-
-    // Leave the window
-    cy.get("[data-test=cancel]").click();
-  });
-
-  it("If the password is updated, the new password is activated immediately", function() {
-    cy.get("[data-test=edit-user-mstein]").should("be.visible");
-
-    // User enters correct password
-    cy.get("[data-test=edit-user-mstein]").click();
-    cy.get("[data-test=user-password-textfield] input").type("test");
-
-    // User enters new password and confirms it
-    cy.get("[data-test=new-password-textfield] input").type("test2");
-    cy.get("[data-test=new-password-confirmation-textfield] input").type("test2");
-    cy.get("[data-test=submit]").click();
-
-    // A success snackbar is displayed
-    cy.get("#client-snackbar")
-      .should("be.visible")
-      .contains("Password successfully changed");
-
-    // The user table should be visible again
-    cy.get("[data-test=userdashboard]").should("be.visible");
-
-    // Edit user password again
-    cy.get("[data-test=edit-user-mstein]").click();
-
-    // User enters old (wrong) password
-    cy.get("[data-test=user-password-textfield] input").type("test");
-
-    // User enters new password and confirmation correctly
-    cy.get("[data-test=new-password-textfield] input").type("test");
-    cy.get("[data-test=new-password-confirmation-textfield] input").type("test");
-    cy.get("[data-test=submit]").click();
-
-    // The warning "Incorrect password" is displayed
-    cy.get("#userPassword-helper-text").contains("Incorrect password");
-
-    // User enters now the new password
-    cy.get("[data-test=user-password-textfield] input").clear();
-    cy.get("[data-test=user-password-textfield] input").type("test2");
-    cy.get("[data-test=submit]").click();
-
-    // A success snackbar is displayed
-    cy.get("#client-snackbar")
-      .should("be.visible")
-      .contains("Password successfully changed");
-  });
-});
+const testUserName = "passwordChangeUser";
+const testUserNamePassword = "test1234";
+
+describe("Users/Groups Dashboard", function() {
+  before(() => {
+    cy.login("root", "root-secret");
```

Comment:
I would outsource the root-secret into a variable

Reply:
this is part of a bigger problem with e2e tests and will be handled separately